components/org.wso2.carbon.consent.mgt.core/src/main/java/org/wso2/carbon/consent/mgt/core/dao/impl/ReceiptDAOImpl.java

```diff
@@ -148,20 +167,26 @@ public Receipt getReceipt(String receiptId) throws ConsentManagementException {
                 service = service.replaceAll(QUERY_FILTER_STRING_ANY_ESCAPED, SQL_FILTER_STRING_ANY);
             }
 
+            if (state == null) {
```

Comment:
Can we use stringutills here?

Reply:
Just want to check null here. 
IMO stringUtils should use when we want to check empty as well.